assets/css/app.scss

```diff
@@ -68,24 +70,25 @@ $z-properties-panel-context: (
 @import "properties_panel/tab_panels";
 @import "properties_panel/vehicle_properties_panel";
 @import "recent_searches";
+@import "route_filter";
 @import "route_ladder";
 @import "route_ladders";
-@import "route_filter";
 @import "route_picker";
 @import "route_pill";
+@import "schedule_adherence";
 @import "search_form";
 @import "search_page";
 @import "search_results";
 @import "settings_page";
 @import "shuttle_map";
-@import "shuttle_picker";
 @import "shuttle_picker_container";
+@import "shuttle_picker";
 @import "stop_card";
 @import "street_view_button";
 @import "swings_view";
-@import "vehicle_card";
 @import "vehicle_icon";
 @import "vehicle_map";
+@import "vehicle_properties_card";
```

Comment:
**praise:** not going to select the thole thing, but thank you for sorting this!